src/firefly/java/edu/caltech/ipac/firefly/server/query/ibe/IbeQueryArtifact.java

```diff
@@ -229,6 +268,63 @@ private static RelatedData makeWiseRelatedDataItem(WorldPt wp, String sizeInDeg,
         return RelatedData.makeTabularRelatedData(params, dataKey, desc);
     }
 
+    public static  List<RelatedData> get2MassRelatedData(WorldPt wp, String sizeInDeg) {
+        List<RelatedData> rdList= new ArrayList<>();
+        rdList.add(get2MassRelatedDataItem(wp,sizeInDeg,"glint","glint_arti","2MASS Glints Artifacts"));
+        rdList.add(get2MassRelatedDataItem(wp,sizeInDeg,"pers","pers_arti","2MASS Persistence Artifacts"));
+        return rdList;
+    }
+
+    private File getWiseScanIdArtifact(final TableServerRequest req) throws IOException, DataAccessException {
+
+       WiseRequest sreq= new WiseRequest();
+       sreq.setParam(QueryIBE.MISSION, WiseIbeDataSource.WISE);
+       sreq.setParam("scanid", req.getParam("scan_id"));
+       sreq.setParam("band", req.getParam("band"));
+       sreq.setParam("subsize", req.getParam("subsize"));
+       sreq.setParam(WiseRequest.PRODUCT_LEVEL, "1b");
+       sreq.setSchema(WiseRequest.MERGE);
+
+       try {
+           DataObject row = queryIBE(sreq);
+           //Step 2: modify TableServerRequest
+           if (row != null) {
+               sreq.setParams(req.getParams());
+               sreq.setParams(IpacTableUtil.asMap(row));
+
+               return getIBEData(sreq);
+           }
+       } catch (Exception e) {
+           // some may not have artifacts
+       }
+       return null;
+   }
+
+    private File getWiseCoaddIdArtifact(final TableServerRequest req) throws IOException, DataAccessException {
```

Comment:
I don't think you are using this function, If you want to keep it then you should document why.

Reply:
you are right. I made these  functions in case they might be added into the `getFinderChartArtifact` (line 71). I don't think it apply to FinderChart case, since it's position search only. I should just take these function out?